ocaml/xapi-idl/storage/storage_interface.ml

```diff
@@ -362,6 +362,11 @@ module Errors = struct
     | No_storage_plugin_for_sr of string
     | Content_ids_do_not_match of (string * string)
     | Missing_configuration_parameter of string
+    | Migration_preparation_failure of string
+    | Migration_mirror_fd_failure of string
```

Comment:
Can the errors that are exclusive to smapiv1 be doc-commented with that information?